src/app/Main/Body/QueryPage/Results/TableResults/TableContent-Rows.component.tsx

```diff
@@ -9,28 +9,30 @@ function chooseAlignment(element: string): "left" | "right" {
     return "right";
 }
 
-export function RowValues(
+interface TableRowInterface {

```

Comment:
reject: TableRowProps?

Reply:
Yes, it was changed in the commit "Unify names of interfaces".